frontend/benefit/handler/src/components/newApplication/formContent/companySection/deMinimisAid/DeMinimisAidForm.tsx

```diff
@@ -0,0 +1,143 @@
+      columns={10}
+      css={`
+        margin-bottom: ${theme.spacing.s};
+        margin-top: ${theme.spacing.m};
+        margin-left: ${theme.spacing.m};
+      `}
+    >
+      <$GridCell
+        $colSpan={8}
+        as={$Grid}
+        columns={8}
+        bgColor
+        bgHorizontalPadding
+        bgVerticalPadding
+      >
+        <$GridCell $colSpan={4}>
+          <TextInput
+            id={fields.granter.name}
+            name={fields.granter.name}
+            label={fields.granter.label}
+            placeholder={fields.granter.placeholder}
+            onBlur={formik.handleBlur}
+            onChange={formik.handleChange}
+            value={formik.values.granter}
+            invalid={!!getErrorMessage(DE_MINIMIS_AID_KEYS.GRANTER)}
+            aria-invalid={!!getErrorMessage(DE_MINIMIS_AID_KEYS.GRANTER)}
+            errorText={getErrorMessage(DE_MINIMIS_AID_KEYS.GRANTER)}
+            required
+          />
+        </$GridCell>
+        <$GridCell $colSpan={2}>
+          <TextInput
+            id={fields.amount.name}
+            name={fields.amount.name}
+            label={fields.amount.label || ''}
+            placeholder={fields.amount.placeholder}
+            onBlur={formik.handleBlur}
+            onChange={(e) =>
+              formik.setFieldValue(
+                fields.amount.name,
+                stringFloatToFixed(e.target.value)
+              )
+            }
+            value={formatStringFloatValue(formik.values.amount)}
+            invalid={!!getErrorMessage(DE_MINIMIS_AID_KEYS.AMOUNT)}
+            aria-invalid={!!getErrorMessage(DE_MINIMIS_AID_KEYS.AMOUNT)}
+            errorText={getErrorMessage(DE_MINIMIS_AID_KEYS.AMOUNT)}
+            required
+          />
+        </$GridCell>
+        <$GridCell $colSpan={2}>
+          <DateInput
+            id={fields.grantedAt.name}
+            name={fields.grantedAt.name}
+            label={fields.grantedAt.label}
+            placeholder={fields.grantedAt.placeholder}
+            language={language}
+            onBlur={formik.handleBlur}
+            onChange={(value) =>
+              formik.setFieldValue(fields.grantedAt.name, value)
+            }
+            value={formik.values.grantedAt}
+            invalid={!!getErrorMessage(DE_MINIMIS_AID_KEYS.GRANTED_AT)}
+            aria-invalid={!!getErrorMessage(DE_MINIMIS_AID_KEYS.GRANTED_AT)}
+            errorText={getErrorMessage(DE_MINIMIS_AID_KEYS.GRANTED_AT)}
+            maxDate={DE_MINIMIS_AID_GRANTED_AT_MAX_DATE}
+            required
+          />
+        </$GridCell>
+      </$GridCell>
+      <$GridCell
+        $colSpan={2}
+        css={`
+          padding-top: 25px;
+          padding-left: ${theme.spacing.s};
+        `}
+      >
+        <Button
+          theme="coat"
+          disabled={
+            !(
+              formik.values.granter &&
+              formik.values.amount &&
+              formik.values.grantedAt
+            ) ||
+            !formik.isValid ||
```

Comment:
Isn't formik.isValid stronger than those listed above?

Reply:
Added this just from the applicant side. The whole deminimis things are 90% identical on both sides, and it feels illegal. Maybe refactor some day.